server/src/controllers/lineController.ts

```diff
@@ -9,15 +9,9 @@ class LineController {
   }
 
   async getNextCustomer(req: Request, res: Response): Promise<void> {
-    const serviceIds: number[] = req.body.service_ids;
   
-    if (!serviceIds || !Array.isArray(serviceIds) || serviceIds.length === 0 || !serviceIds.every(id => typeof id === 'number' && id > 0 && Number.isInteger(id))) {
-      res.status(400).send({ error: 'Invalid service_ids' });
-      return;
-    }
-
     try {
-      const nextCustomerId = await this.lineDAO.getNextCustomer(serviceIds);
+      const nextCustomerId = await this.lineDAO.getNextCustomer();
```

Comment:
I still don't know how did you remove the serviceId for getNextCustomer(); ( Indeed => this method is related about the calling next customer? if it is, the name should be more related, coz we can think that this function is responsible for creating sth like ticket number, not calling that )